docs/general/dApps.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 2
+sidebar_label: 'dApps'
```

Comment:
Is it better to use `dApp` instead of `dApps` to be more precise?

Reply:
ok, updated